builtin/mainmenu/dlg_create_world.lua

```diff
@@ -15,11 +15,39 @@
 --with this program; if not, write to the Free Software Foundation, Inc.,
 --51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
+local worldname = ""
+
 local function create_world_formspec(dialogdata)
 	local mapgens = core.get_mapgen_names()
 
 	local current_seed = core.settings:get("fixed_map_seed") or ""
 	local current_mg   = core.settings:get("mg_name")
+	local gameid = core.settings:get("menu_last_game")
+
+	local game, gameidx = nil , 0
+	if gameid ~= nil then
+		game, gameidx = gamemgr.find_by_gameid(gameid)
+		
+		if gameidx == nil then
+			gameidx = 0
+		end
+	end
+
+	local gamepath = minetest.get_game(gameidx).path
+	local gameconfig = Settings(gamepath.."/game.conf")
+
+	local disallowed_mapgens = (gameconfig:get("disallowed_mapgens") or ""):split()
+	for key, value in pairs(disallowed_mapgens) do
+		disallowed_mapgens[key] = value:trim()
+	end
+
+	if disallowed_mapgens then
```

Comment:
Superfluous check. `string.split` never returns `nil`.

Reply:
rubenwardy requested it be this way. I know his commented version is a little different, but he asked me to adjust to this, so will see what he says.

Reply:
I didn't notice this aha, krock is right unfortunately. But doesn't matter too much